src/coreclr/CMakeLists.txt

```diff
@@ -250,8 +250,10 @@ if(NOT CLR_CMAKE_HOST_MACCATALYST AND NOT CLR_CMAKE_HOST_IOS AND NOT CLR_CMAKE_H
   add_subdirectory(utilcode)
   add_subdirectory(inc)
 
-  add_subdirectory(ilasm)
-  add_subdirectory(ildasm)
+  if (NOT CLR_CMAKE_TARGET_ANDROID)
```

Comment:
Could we instead define a property for "build executables" instead of per-platform?
And also define one for the case above for "NativeAOT-only"?